packages/svelte/src/constants.ts

```diff
@@ -0,0 +1,8 @@
+// TODO: we might want to call this ui.svelte.init instead because
+// it doesn't only track mounting time (there's no before-/afterMount)
+// but component init to mount time.
+export const UI_SVELTE_MOUNT = 'ui.svelte.mount';
```

Comment:
I am always in favor of calling things the way users are familiar with. If "initializing" is a term that is used in the svelte community for mounting components, I vote we call it `ui.svelte.init`

Reply:
My train of thought for maybe renaming this to init instead of mount is that this span does not only track mounting time. Because the `onMount` hook is the only lifecycle hook we have for mounting, we only know when mounting is completed but not when it is started. What we do know, however, is when we start _initializing_ the component (i.e. when the <script> block of a component is executed). So this span tracks exactly this duration: From the beginning of script execution until the component is completely mounted in the DOM. Which technically makes this more an _initialization_ span than a _mounting_ span.

Reply:
I renamed it to init. Makes more sense IMO. Let's go with that